### DISCUSSION TITLE
Invalid email address on sign up

### DISCUSSION THREAD

**Discussion by zerrium** [vercel/community#118] [Help] [answered]
I cant sign up with my github account (with @mail.com email address). It says invalid email address. Any solution for this?

**Comment by markozxuu** [comment]
Could you help with this 🙏  cc @styfle

**Comment by styfle** [comment, marked as answer]
Hi @zerrium 

Your email address might have been flagged as possible spam and blocked from sign up. 

Please reach out at https://vercel.com/support and we’ll get it sorted out, thanks!

**Reply by magikadev** [reply]
I got same error.

I already sent an email to support@vercel.com
